src/engine/server_iv.c

```diff
@@ -1128,14 +1055,79 @@ iv_op(struct ds_iv_ns *ns, struct ds_iv_key *key, d_sg_list_t *value,
 		 */
 		D_WARN("retry upon %d for class %d opc %d\n", rc,
 		       key->class_id, opc);
-		/* Yield to avoid hijack the cycle if IV RPC is not sent */
-		ABT_thread_yield();
+		/* sleep 1sec and retry */
+		dss_sleep(1000);
 		goto retry;
 	}
 
 	return rc;
 }
 
+static void
+iv_op_ult(void *arg)
+{
+	struct iv_op_ult_arg *ult_arg = arg;
+
+	D_ASSERT(ult_arg->iv_sync.ivs_mode == CRT_IV_SYNC_LAZY);
+	/* Since it will put LAZY sync in a separate and asynchronous ULT, so
+	 * let's use EAGER mode in CRT to make it simipler.
+	 */
+	ult_arg->iv_sync.ivs_mode = CRT_IV_SYNC_EAGER;
```

Comment:
I don't have enough knowledge on the implementation of EAGER and LAZY modes in Cart IV, but this change looks to me like CRT_IV_SYNC_LAZY will be useless in Cart IV?

Reply:
yes, daos will not use async cart IV for now.